### PR TITLE
Update information for release 0.15.0

### PR DESCRIPTION
**Pull request by skoudoro**
Hi @arokem, @Garyfallidis,

Can you check this PR for the Release 0.15.0?

Thank you

### CODE REVIEW

File: dipy/info.py [hunk 1]
```diff
@@ -72,9 +72,7 @@
 Please see the LICENSE file in the dipy distribution.
 
 DIPY uses other libraries also licensed under the BSD or the
-MIT licenses, with the only exception of the SHORE module which
-optionally uses the cvxopt library. Cvxopt is licensed
```
Comment (arokem) [inline dipy/info.py]: 🎉

File: doc/api_changes.rst [hunk 2]
```diff
@@ -5,6 +5,29 @@ API changes
...
+-----------------
+
+**IO**
+
+``load_tck`` and ``save_tck`` from ``dipy.io.streamline`` has been added. They are highly recommended for managing streamlines.
+
+**Gradient Table**
+
+The default value of ``b0_thresold`` has been changed(from 0 to 50). This change can impact your algorithm.
+If you want to go back to your previous result, make sure to set up this optional argument to 0.
```
Comment (arokem) [inline doc/api_changes.rst]: I would write this latter as: "If you want to assure that your code runs in exactly the same manner as before, please initialize your gradient table with the keyword argument `b0_threshold` set to 0."

File: doc/api_changes.rst [hunk 3]
```diff
@@ -5,6 +5,29 @@ API changes
...
+
+**Visualization**
+
+``dipy.viz.fvtk`` module has been removed. Use ``dipy.viz.*`` instead. This implies the following important changes:
+- Use ``from dipy.viz import window, actor`` instead of ``from dipy.viz import fvtk`.
+- Use ``window.Renderer()`` instead of ``fvtk.ren()``.
+- All available actors are in ``dipy.viz.actor`` instead of ``dipy.fvtk.actor``.
+- UI elements are available in ``dipy.viz.ui``.
+
+``dipy.viz`` depends on FURY package. To get more informations about FURY, got to https://fury.gl
```
Comment (arokem) [inline doc/api_changes.rst]: Typo, "got to"=> "go to"

**Comment by arokem** [comment]
Overall LGTM. I had just a couple of small comments/suggestions.

**Comment by arokem** [comment]
Exciting!

**Comment by skoudoro** [comment]
Thank you for the feedback!

**Pull request merged**